tests/coverage_analysis.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
 
 set -e
-
 readonly libpath="${LIBPATH:-../guibot}"
 readonly coverage="${COVERAGE:-coverage3}"
 readonly submit="${SUBMIT:-0}"
 
+echo " ---- running tests with $coverage ---- "
```

Comment:
This has extra spaces around the comment dashes unlike the headers above - should we simply use the same header format as in the other scripts?